tsconfig.json

```diff
@@ -7,9 +7,9 @@
     "jsxFactory": "h",
     "jsxFragmentFactory": "Fragment",
     "lib": ["esnext", "dom"],
-    "module": "es6",
+    "module": "esnext",
```

Comment:
Any specific reason to change from `es6` to `esnext` here?

Reply:
Apparently yes, this is due to the need to use dynamic imports in `StepComponentMap.ts`. Is this a deal-breaker? Is there a way around it?

Reply:
@ethan605 I've updated the CHANGELOG with this info **only**. Will all additional information regarding this module in specific be added on this file as well? i.e. separate bundle builds, auth flow explanation and screens added, etc.